app/register-organization/page.tsx

```diff
@@ -0,0 +1,283 @@
+  const onSubmit = async (formValues: OrganizationFormDetail) => {
+    try {
+      const res = await createOrganization(formValues).unwrap();
+      if (res?.data) {
+        toast.success(res?.message);
+        form.reset();
+        setTimeout(() => {
+          router.replace(ROUTES.DASHBOARD);
+        }, 500);
+      }
+    } catch (err: any) {
+      toast.error(err?.data?.message);
+    }
+  };
+
+  return (
+    <AuthenticationPageTemplate className="w-full">
+      <Form {...form}>
+        <form
+          onSubmit={form.handleSubmit(onSubmit)}
+          className="flex flex-col gap-6"
+        >
+          <FormField
+            control={form.control}
+            name={"name"}
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel required>Organization name</FormLabel>
+
+                <FormControl>
+                  <Input placeholder={"name"} type={"string"} {...field} />
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name="size"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel required> Organization Size</FormLabel>
+
+                <Select
+                  onValueChange={field.onChange}
+                  defaultValue={field.value}
+                >
+                  <FormControl>
+                    <SelectTrigger
+                      className={`${field?.value === "" ? "text-placeholder" : "text-text-color"}`}
+                    >
+                      <SelectValue placeholder="select" />
+                    </SelectTrigger>
+                  </FormControl>
+                  <SelectContent>
+                    {ORG_SIZE_OPTIONS?.map((orgSizes, index) => (
+                      <SelectItem key={index} value={orgSizes?.value}>
+                        {orgSizes?.label}
+                      </SelectItem>
+                    ))}
+                  </SelectContent>
+                </Select>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <div className="flex flex-col gap-6 sm:flex-row">
+            <FormField
+              control={form.control}
+              name="country"
+              render={({ field }) => (
+                <FormItem className="flex flex-1 flex-col">
+                  <FormLabel required>Country</FormLabel>
+
+                  <Popover
+                    open={countryPopover}
+                    onOpenChange={setCountryPopover}
+                  >
+                    <PopoverTrigger asChild>
+                      <Button
+                        variant="popover"
+                        className={`${field?.value === "" ? "text-placeholder" : "text-textColor"}`}
+                        role="combobox"
+                      >
+                        <span className="line-clamp-1 text-left">
```

Comment:
if using text-left is necessary only then use it else remove it because normally text is left aligned

Reply:
necessary here.